src/main/java/org/ays/auth/model/request/AysUserCreateRequest.java

```diff
@@ -16,10 +16,13 @@
 
 /**
  * Represents a request for creating a new user in the system.
- * This class contains all the necessary details required to create a user, including their personal information,
- * contact details, and roles. The fields are validated to ensure they meet the required constraints.
+ * This class contains all the necessary details required to create a user,
+ * including their personal information,
+ * contact details, and roles. The fields are validated to ensure they meet the
+ * required constraints.
  * <p>
- * This class uses various annotations for validation to ensure the data integrity and consistency.
+ * This class uses various annotations for validation to ensure the data
+ * integrity and consistency.
```

Comment:
Özel bir formatter kullanıyosun sanırım, herkes default IntellIJ formatter'ını kullanıyor bu sebeple burayı eski hâline alabiliriz
```suggestion
 * This class uses various annotations for validation to ensure the data integrity and consistency.
```